test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -125,3 +126,15 @@ def edit_community(self, new_community_name: str, new_community_description: str
 
     def go_back_to_community(self):
         click_obj_by_name(CommunitySettingsComponents.BACK_TO_COMMUNITY_BUTTON.value)
+
+    def delete_current_community_channel(self):
+        MainUi.wait_for_banner_to_disappear()
```

Comment:
I am not sure if this is needed :thinking:  `click_obj_by_name` will wait for the object to become visible and that could only happen if the banner disappears. Just ensure `CHAT_MORE_OPTIONS_BUTTON`, etc. have `visible` attribute set to true.

Reply:
I added it only after struggling to understand why the click works only sometimes and found that the banner animation was to blame. Also, the step is done near the time banners disappear.

> Just ensure CHAT_MORE_OPTIONS_BUTTON, etc. have visible attribute set to true.

It has. However, this is not mutually exclusive with the banner. All the app workflows are available when the banner spans.